lib/std/os/linux.zig

```diff
@@ -635,7 +635,8 @@ pub fn futex2_waitv(
 }
 
 /// Wait on a futex.
-/// Identical to `FUTEX.FUTEX_WAIT_BITSET`, except it is part of the futex2 family of calls.
+/// Identical to the traditional FUTEX_WAIT_BITSET op, except it is part of the
```

Comment:
Nit: Please keep the backticks (`futex2` below should really use them as well).

Reply:
I've restored them. Is it just a stylistic choice or do they serve some kind of purpose? I'm asking out of curiosity.

Reply:
Doc comments currently use a subset of Markdown.

Reply:
Oh right, that makes sense. Thanks